.jenkins/job_beam_PreCommit_Website_Stage.groovy

```diff
@@ -0,0 +1,66 @@
+import common_job_properties
```

Comment:
I've had bad experiences with submodules relevant to the article I posted there -- if you really want to use them I'll give it a shot, but I think the downsides are way larger than having these two (eventually three) files live in a different repository.